enterprise/web.core.syntax/src/org/netbeans/modules/web/core/syntax/ErrorAnnotationImpl.java

```diff
@@ -151,9 +149,9 @@ public void run() {
     
     /** Transforms ErrosInfo to Annotation
      */
-    private Collection getAnnotations(ErrorInfo[] errors, StyledDocument document) {
+    private Collection<LineSetAnnotation> getAnnotations(ErrorInfo[] errors, StyledDocument document) {
         BaseDocument doc = (BaseDocument) document;
-        HashMap map = new HashMap(errors.length);
+        Map<Integer, LineSetAnnotation>  map = new HashMap(errors.length);
```

Comment:
i overlooked this one first round. Diamond on the right.

Reply:
done..

Reply:
fixed..